source/reference/beta-program.txt

```diff
@@ -73,15 +73,15 @@ Beta Features
    * - :ref:`c2c-beta-document-filtering`
      - .. include:: /includes/document-filtering-intro.rst
 
-   * - :ref:`c2c-beta-orr`
-     - # DESCRIPTION TBD 
-
    * - :ref:`c2c-beta-destination-data-handling`
      - .. include:: /includes/destinationDataHandling-introduction.rst
 
    * - :ref:`c2c-beta-many-to-one`
      - .. include:: /includes/many-with-one-cluster.rst
 
+   * - :ref:`c2c-beta-orr`
+     - .. include:: /includes/orr-intro.rst
```

Comment:
(question / suggestion)
What is your "organizational strategy" for this table? If featured are just listed in random order, maybe alphabetize, at minimum?

Reply:
Hi! This is intended to be alphabetized — the changes made in #332 changes the name of some of these features in the table, so I didn't want to change them here to avoid conflicting changes! For instance, "Synchronize Data Between Multiple Source Clusters Simultaneously" will become "Many-to-One Migrations". Also ABC migrations will be added to the top.

Reply:
Thanks @ajhuh-mdb! This is great!